.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Run Tests
```

Comment:
If we want to run tests by github actions I think that it is worth to remove/disable that in Travis CI https://travis-ci.org/github/backtrace-labs/backtrace-android

Reply:
We can fully replace Travis CI. [GitHub Actions is free for private repos](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions). Do you agree @vlussenburg to commit to migrating fully to GitHub Actions and deprecating Travis?

Reply:
Yes

Reply:
Sounds good! I'll make the README badge change here in this pull request as well